app/src/main/java/com/github/gotify/NotificationSupport.java

```diff
@@ -1,7 +1,9 @@
 package com.github.gotify;
 
 import android.app.NotificationChannel;
```

Comment:
It is possible to change this via the app itself [(see here)](https://stackoverflow.com/questions/55717770/cant-update-sound-programmatically-for-notification-channel-on-android-oreo)

Maybe we should do it this way instead of having two distinct versions?

Reply:
Also: Should we really create 4 channels for each priority? I think we can just use one channel and then use the notifications priority.

True, that reduces the flexibility a bit, but to be honest: we are kinda "abusing" the channel system a bit. I think that this makes more sense than the current implementation as it aligns with the original meaning of priority

Reply:
> also it doesnt actually break anything, you just need to manually change it for each app

I see this as breaking. When a user has a lot of apps, he now has to configure lots of them when he only wants to change the behavior of all high priority messages.

Reply:
What about making that feature a switch so users can decide if they prefer the app or priority based channels? Depending on the user's choice, the channels will then be added and the others removed.

Reply:
Yes, this is what I meant with feature flag. Sorry, that I didn't write this clearly. I think a setting beside the theme switch should be added to switch between priority based & app based channels.

Reply:
@newhinton do you think that can be done?